spark-jobs/src/main/scala/za/co/absa/enceladus/standardization/StandardizationPropertiesProvider.scala

```diff
@@ -81,7 +81,7 @@ class StandardizationPropertiesProvider {
   }
 
   private def getXmlOptions[T](cmd: StandardizationConfigParser[T]): HashMap[String, Option[RawFormatParameter]] = {
-    if (cmd.rawFormat.equalsIgnoreCase("xml")) {
```

Comment:
Why do these changes?

Reply:
Noticed this by chance. Since this had been created, the `rawFormat` begun to be normalized upon `cmd: Config` creation.